slimevr-setup.md

```diff
@@ -78,43 +78,55 @@ If any tracker displays ERROR as it's status, or have orange and blue light perm
-1. You can mount trackers in any place on the designated body part you find comfortable. Make sure the tracker moves when you bend the joint to register movement. **Pay special attention to the waist tracker, there are many places where you can mount it and it won't register you bending over.**
+1. You can mount trackers tiled forward/backward or on the side, this will not mess up the tracking. 
+1. You can mount trackers in any place on the designated body part you find comfortable, however the less the area is deformed by muscular movement the better the tracking will be. Make sure the tracker moves when you bend the joint to register movement. **Pay special attention to the waist tracker, there are many places where you can mount it and it won't register you bending over.**
+
 
 ### Recommended mounting points
 
 ![img](https://eiren.cat/ECvD)
 
 ![img](https://eiren.cat/DvJi)
 
+#### Alternative mounting points
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MMJ8843zqNM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 Recommended tracker positions are:
 
-* Waist: facing left above waist.
+* Waist: facing left above waist or in the centre of the waist depending on your build.
 * Chest: facing forward middle of the chest (or facing left the same as waist, whatever is more comfortable). Pay attention to tracker extension "front" and "up" sides (see picture below).
-* Leg: right above the knee.
+* Leg: right above the knee or higher up the thigh depending on your build.
 * Ankle: on the ankle.
 * Foot: top of the foot. Tracker's "up" facing towards ankle, and "front" facing up.
 
-**Updated recommendation**: Put main tracker on the chest, and tracker extension on the waist. It's more comfortable and stable this way, since your hand won't bump into waist tracker.
+You can also put the main tracker on the chest, and tracker extension on the waist. It's more comfortable and stable this way, since your hand won't bump into waist tracker.
+
 
 ![img](https://eiren.cat/TyTd)
 *Form and direction of axes are subject to change in future revisions!*
 
 ## Configure proportions and trackers
 
-Make sure that all your trackers are active in SlimeVR before proceeding. Launch SteamVR when you're ready.
+Before launching SteamVR, check the checkboxes relative to how many SteamVR virtual trackers you wish to spawn.
+
+![img](https://i.imgur.com/To9z4Pd.png)
+
+Please note that this only affects the virtual trackers in Steam and not your SlimeVR trackers. If you only have Waist and Legs ticked, they will still be accessing up to an 8 IMU setup (waist,chest,legs,ankles and feet). Additionally, VRChat only supports 3 virtual trackers so it does not require more than Waist and Legs.
```

Comment:
> Additionally, VRChat only supports 3 virtual trackers

I often say "unmodded VRChat" being sneaky that people can actually mod... maybe this will make devs integrate iktweaks lol

Reply:
Yeah! My only thought is with our online docs to make no suggestions to breaking ToS, as I didn't want to risk anything